packages/flutter/lib/src/material/progress_indicator.dart

```diff
@@ -312,6 +312,9 @@ class _LinearProgressIndicatorPainter extends CustomPainter {
 ///
 /// {@tool dartpad}
 /// This example showcases determinate and indeterminate [LinearProgressIndicator]s.
+/// The [LinearProgressIndicator]s will use the updated Material 3 Design appearance
```

Comment:
maybe we can use something like:
```suggestion
/// The [LinearProgressIndicator]s will use the [updated Material 3 Design appearance](https://m3.material.io/components/progress-indicators/overview)
```